src/components/AudioPlayer/AudioPlayer.js

```diff
@@ -23,7 +27,9 @@ class AudioPlayerSub extends Component {
 
     TrackPlayer.updateOptions({
       // Whether the player should stop running when the app is closed on Android
-      stopWithApp: true,
+      android: {
+        appKilledPlaybackBehavior: AppKilledPlaybackBehavior.ContinuePlayback,
+      },
```

Comment:
Is this the correct equivalent?

> stopWithApp turns into stoppingAppPausesPlayback

https://github.com/doublesymmetry/react-native-track-player/blob/main/CHANGELOG.md#300-110822

Reply:
Probably not, I had just followed their migration guide and wanted to get straight to testing builds. I can change it now though.